aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -117,10 +117,12 @@ The hosted service is activated once at app startup and gracefully shut down at
 
 ## BackgroundService
 
-`BackgroundService` is a base class for implementing a long running <xref:Microsoft.Extensions.Hosting.IHostedService>. `BackgroundService` defines two methods for background operations:
+`BackgroundService` is a base class for implementing a long running <xref:Microsoft.Extensions.Hosting.IHostedService>. `BackgroundService` provides the `ExecuteAsync(CancellationToken stoppingToken)` abstract method to contain the service's logic. The `stoppingToken` is triggered when [IHostedService.StopAsync](xref:Microsoft.Extensions.Hosting.IHostedService.StopAsync*) is called. The implementation of this method returns a `Task` that represents the entire lifetime of the background service.
```

Comment:
Note ExecuteAsync should not synchronously block or it may block other services from starting.